pyproject.toml

```diff
@@ -2,31 +2,50 @@
 requires = ["setuptools", "setuptools-scm"]
 build-backend = "setuptools.build_meta"
 
+[tools.setuptools]
+packages = ["colorizer_data"]
+
+[tool.setuptools.packages.find]
+where = ["."]
+include = ["colorizer_data"]
+namespaces = false
+
 [project]
-name = "timelapse-colorizer-data"
+name = "colorizer_data"
 authors = [
     {name = "", email = "danielt@alleninstitute.org"},
+    {name = "", email = "peyton.lee@alleninstitute.org"},
 ]
-description = "data prep scripts"
-# readme = ""
+description = "Utilities to convert data for viewing in Timelapse Colorizer"
+readme = "README.md"
 requires-python = ">=3.8"
-keywords = ["one", "two"]
 license = {text = "BSD-3-Clause"}
 classifiers = [
     "Programming Language :: Python :: 3",
 ]
 dependencies = [
-    "aicsimageio[czi]>=4.11.0",
-    "aicspylibczi>=3.0.5",
     "numpy",
     "pandas",
     "pillow",
     "scikit-image",
 ]
 dynamic = ["version"]
 
+[project.optional-dependencies]
+# To install, run `pip install -e '.[dev]'`
+dev = [  
+    'bump-my-version', 
+]
+
+[project.urls]
+Homepage = "https://github.com/allen-cell-animated/colorizer-data"
+Issues = "https://github.com/allen-cell-animated/colorizer-data/issues"
+
 [project.scripts]
-my-script = "my_package.module:function"
+convert_nucmorph_data = "colorizer_data.bin.example_scripts.convert_nucmorph_data:main"
+convert_emt_migration_data = "colorizer_data.bin.example_scripts.convert_emt_migration_data:main"
+convert_emt_nuclear_data = "colorizer_data.bin.example_scripts.convert_emt_nuclear_data:main"
+convert_emt_h2b_labelfree = "colorizer_data.bin.example_scripts.convert_emt_h2b_labelfree:main"
```

Comment:
In general, if the example scripts are installed with this package, then their dependencies should be dependencies here.  I'm not sure if there is a normal counterexample for this, but you need to at least document why after installing this package, a bin script would fail.  Maybe either in the README or a comment at the top of the example scripts.

Reply:
something like "WARNING: the example scripts include external dependencies which you will have to install manually as they are not considered proper dependencies of this package"

Reply:
Done, thank you!